libs/indibase/inditelescope.cpp

```diff
@@ -1709,9 +1749,16 @@ bool Telescope::updateTime(ln_date *utc, double utc_offset)
 
 bool Telescope::updateLocation(double latitude, double longitude, double elevation)
 {
-    INDI_UNUSED(latitude);
-    INDI_UNUSED(longitude);
+    // FIXME (sterne-jaeger): needs to be called from all classes deriving from inditelescope.
```

Comment:
perhaps a better approach is to define updateObservorLocation() which does this and then call it from ISNewSwitch just after calling updateLocation? This way the derived classes don't need to call this.

Reply:
Ooops, that's a leftover from a first attempt. I first thought I would need it for calculating the pier side - which turned out to be unnecessary. I'll remove it.

Reply:
OK, have to check it. This functionality is pushed down from telescope_simulator.cpp

Reply:
I've implemented it now as proposed. Now we keep the observer's location in the memory for all drivers.